ecommerce/frontend/src/components/messages/Conversations.js

```diff
@@ -0,0 +1,43 @@
+import './Conversations.less'
+import 'react-chat-elements/dist/main.css'
+
+import { useEffect } from 'react'
+
+import { useChatContext } from '../../context/ChatContext'
+import { Conversation } from './Conversation'
+import { ConversationList } from './ConversationList'
+import { Empty } from 'antd'
+
+export const Conversations = () => {
+    const { conversations, getConversations, conversation, setConversation } = useChatContext()
+
+    const onSelectConversation = conversation => {
+        setConversation(conversation)
+    }
+
+    useEffect(() => {
+        async function fetch() {
+            await getConversations()
+        }
+        fetch()
+    }, [])
+
+    if (conversations !== null && !conversations.length) {
+        return (
+            <div className="conversations-empty">
+                <Empty description={'You have no conversations'} />
+            </div>
+        )
+    }
```

Comment:
To enter into if condition and display 'you have no conversations'
Both `conversations is not null` and `conversations.length is 0` true

`if(!conversations or !conversations.length)` could be more error tolerant.

Reply:
Sure, it is just that conversations normally can only take null or array